diff --git a/node_modules/babel-plugin-remove-graphql-queries/index.js b/node_modules/babel-plugin-remove-graphql-queries/index.js
index 386144d..eec79a8 100644
--- a/node_modules/babel-plugin-remove-graphql-queries/index.js
+++ b/node_modules/babel-plugin-remove-graphql-queries/index.js
@@ -189,9 +189,7 @@ function _default({
               const shortResultPath = `public/static/d/${this.queryHash}.json`;
               const resultPath = nodePath.join(process.cwd(), shortResultPath); // Remove query variable since it is useless now
 
-              if (this.templatePath.parentPath.isVariableDeclarator()) {
-                this.templatePath.parentPath.remove();
-              } // only remove the import if its like:
+              // only remove the import if its like:
               // import { useStaticQuery } from 'gatsby'
               // but not if its like:
               // import * as Gatsby from 'gatsby'
